src/lqcut.jl

```diff
@@ -65,10 +57,10 @@ function lq_drift_time_correction(
         σ_t = mvalue(drift_result.σ)
 
         #set cutoff in drift time dimension for later fit
-        t_lower = µ_t - drift_cutoff_sigma * σ_t
-        t_upper = µ_t + drift_cutoff_sigma * σ_t
+        t_lower = µ_t - drift_time_outlier_sigma * σ_t
+        t_upper = µ_t + drift_time_outlier_sigma * σ_t
 
-    elseif mode == :double_gaussian # can be used for detectors with double peaks; not optimized yet
+    elseif ctc_driftime_cutoff_method == :double_gaussian # can be used for detectors with double peaks; not optimized yet
```

Comment:
Maybe also consider here setting `uncertainty=false` if you are only interested in the Best fit values.